cluster-management/setup/cluster-architectures/index.md

```diff
@@ -138,12 +144,13 @@ Look how simple the cloud-config becomes:
 #cloud-config
 
 coreos:
+  etcd2:
+    proxy: on 
+    listen-client-urls: http://localhost:2379
```

Comment:
I was running into that on beta. 

@crawford Are we going to ship 2.0.12 to all channels soon? Customers are going to run into this error otherwise.